packages/button/src/Button.tsx

```diff
@@ -176,26 +176,41 @@ const buttonVariants: { readonly [K in Variant]: string } = {
 const buttonSizes: Record<Size, string> = {
   [Size.XSmall]: css`
     height: 22px;
-    padding: 0 8px;
     font-size: 11px;
     text-transform: uppercase;
     font-weight: bold;
   `,
 
   [Size.Small]: css`
     height: 25px;
-    padding: 0 10px;
     font-size: 14px;
   `,
+
   [Size.Normal]: css`
     height: 32px;
-    padding: 0 12px;
     font-size: 14px;
   `,
 
   [Size.Large]: css`
     height: 45px;
     font-size: 16px;
+  `,
+} as const;
+
+const spanSizes: Record<Size, string> = {
```

Comment:
Should these values conform with our spacing tokens?

Reply:
Probably in the future – This PR shouldn't have any style changes associated with it though.